### PR TITLE
Added wording to clarify retrictions on children of p elements.

### PR DESCRIPTION
**Pull request by adanilo**
Addresses issue #763 for paragraph element description.

### CODE REVIEW

File: sections/semantics-grouping-content.include [hunk 1]
```diff
@@ -165,6 +165,10 @@
       This example still has five structural paragraphs, but now the author can style just the
       <code>div</code> instead of having to consider each part of the example separately.
     </div>
+
+    In general, elements that cannot be children of <{p}> elements includes any elements that
```
Comment (agcolom) [inline sections/semantics-grouping-content.include]: As elements is plural, I think includes should be include (no 's')

**Review by chaals — APPROVED** [review]
with @agcolom's change in it this looks good.

**Comment by simevidas** [comment]
Could you clarify if this means that a `<em style="display: inline-block">` inside a `<p>` is invalid?

**Comment by LJWatson** [comment]
The spec restricts which HTML elements can be used inside the ```<p>``` element. It does not specify which CSS can be applied to those elements.

**Comment by simevidas** [comment]
But the way the note is written implies the use of CSS, doesn’t it?

> In general, elements that cannot be children of p elements include any elements that are inline blocks, inline tables, as well as floated and positioned block-level elements.

Source: [Section 4.4.1.](http://w3c.github.io/html/grouping-content.html#the-p-element)

These scenarios can only be achieved with CSS, not with HTML alone, as far as I know.

**Comment by LJWatson** [comment]
Good point. Do you have suggested wording? If not I'll defer to @adanilo

**Comment by SelenIT** [comment]
For me, it seems that the current wording still implies that CSS can somehow influence on whether these elements can be contained in the paragraph or not. Moreover, it might give the impression to a novice that HTML has the concept of "inline-block elements" and "inline-table elements", in addition to HTML 4 concepts of "block elements" and "inline elements" that are still widely used by many educational resources despite the fact that HTML5 replaced them with the "kinds of content" concept.

Wouldn't it be better to stress the fact that CSS can't change the content model of the element at all, with some wording like this?

> Elements that can't be children of `<p>` element can't be placed inside it regardless their styling, even if they are formatted as inline blocks, inline tables, floated or positioned elements, etc.

**Pull request merged**